frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -269,6 +280,10 @@ const setup = async ({
 };
 
 describe("QueryBuilder", () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
```

Comment:
Moved this to top-level `describe` to avoid duplication. It's needed for new `beforeunload` event unit tests as well.